superset/connectors/sqla/views.py

```diff
@@ -411,6 +411,7 @@ class TableModelView(  # pylint: disable=too-many-ancestors
         "database": QuerySelectField(
             "Database",
             query_func=lambda: db.session.query(models.Database),
+            get_pk_func=lambda a: a.id,
```

Comment:
```suggestion
            get_pk_func=lambda item: item.id,
```

Reply:
I think `item`—which is used elsewhere (including the DAOs)—is likely clearer that `a`.